### PR TITLE
Add RMC sounds, audio objects and asset packs

### PR DESCRIPTION
**Pull request by IntelOrca**

### CODE REVIEW

**Comment by karst** [comment]
The RMC track friction is not using the correct pitch, it should have the same values as the B&M track friction sound.
[auraspecs.rmctest.zip](https://github.com/OpenRCT2/OpenRCT2/files/9790849/auraspecs.rmctest.zip)
Use the following assetpack on current develop version to check the difference. 
At current commit the sound of RMC track friction sounds like mosquito which is sounds quite annoying, while in the assetpack above it sounds more roaring and epic.

I have made a PR on the objects directory to change the sounds to use the correct track friction see right here https://github.com/OpenRCT2/objects/pull/220
Once this is merged the objects repo will need a new release and that should be pushed with this PR.

Lastly the DefaultObjects need to be updated.
In `.h` that is `extern const std::string_view DefaultSelectedObjects[106];` (instead of `103`)
And under `// Music` in `.cpp` you need to add `openrct2.music.acid`, `openrct2.music.dodgems` and `openrct2.music.galaxy`  at line 116.
The three above should also be inserted to `RCT12.cpp` at line 735

That's all I have currently found.

**Comment by karst** [comment]
The only thing I am currently concerned with still is that the music files will be dropped in the build twice rather than the alternative style being able to pull the music from a music object file. Music files are already pretty big as they are and to have them duplicated is kind of a waste of space.
Artifact to show: [OpenRCT2 OpenMusic Objects.zip](https://github.com/OpenRCT2/OpenRCT2/files/9860202/OpenRCT2.OpenMusic.Objects.4.zip)

File: src/openrct2-ui/CMakeLists.txt [hunk 1]
```diff
@@ -232,7 +232,21 @@ if(MACOS_BUNDLE)
         ZIP_URL ${TITLE_SEQUENCE_URL}
         SHA1 ${TITLE_SEQUENCE_SHA1}
     )
-    
+
+    # download opensfx and openmsx
```
Comment (Gymnasiast) [inline src/openrct2-ui/CMakeLists.txt:236]: This only adds downloads to the macOS bundle.

I tried adding counterparts for Linux to `<root>/CMakeLists.txt`, but I’m running into problems there - it seems that merging three `object` folders does not work as intended.

**Review by Gymnasiast — CHANGES_REQUESTED** [review]
Tested this out. The asset download does not seem to work, but installing them manually, I get the lovely Hbyrid RC sounds. So we just have to sort out that bit, I imagine.

**Comment by IntelOrca** [comment]
> Tested this out. The asset download does not seem to work, but installing them manually, I get the lovely Hbyrid RC sounds. So we just have to sort out that bit, I imagine.

try now

**Review by Gymnasiast — APPROVED** [review]

**Comment by karst** [comment]
Changelog entry would probably look best like this:
```
- Feature: [#18312, objects#220, OpenSFX#13] New sound effects for the Hybrid RC & Single Rail RC.
- Feature: [objects#221] Adopt two of Couger's jungle walls as official objects.
- Feature: [OpenSFX#11, OpenMusic#19] Official replacement assetpacks for sound effects & music. 
- Feature: [OpenMusic#14, OpenMusic#15, OpenMusic#18] New music is now available: Galaxy style, Acid style, Dodgems style.
```

**Pull request merged**